library/percona

```diff
@@ -12,9 +12,9 @@ GitCommit: 65855912b9946093cee42f987336440972405391
 Directory: percona-server.80
 File: Dockerfile
 
-Tags: 5.7.30-centos, 5.7-centos, 5-centos, centos, 5.7.30, 5.7, 5, ps-5.7.30, ps-5.7, ps-5, latest
+Tags: 5.7.31-centos, 5.7-centos, 5-centos, centos, 5.7.31, 5.7, 5, ps-5.7.31, ps-5.7, ps-5, latest
 GitCommit: 722aecd7714aa6c2fa3e50ccaef1aef45b1ca5c1
```

Comment:
Looks like the `GitCommit` needs to be updated as well.

Reply:
Thank you, missed this part in late night, commit updated